shared/feature/feedings/src/main/java/com/epmedu/animeal/feedings/presentation/FeedingsScreenUi.kt

```diff
@@ -116,15 +121,40 @@ private fun LoadingState() {
 }
 
 @Composable
-private fun EmptyState() {
+private fun EmptyState(isPendingTab: Boolean) {
     Box(
-        modifier = Modifier.fillMaxSize()
+        modifier = Modifier.fillMaxSize(),
+        contentAlignment = Alignment.Center
     ) {
-        Text(
-            text = stringResource(R.string.no_feedings),
-            modifier = Modifier.align(Alignment.Center),
-            textAlign = TextAlign.Center,
-        )
+        Column(
```

Comment:
I just realised that we can probably reuse this part from `ComingSoonStub` and `ThankYouContent`. What do you think?

Reply:
Sounds good, let's make a common composable.